pkg/apis/component/v1alpha2/capability_types.go

```diff
@@ -51,11 +64,11 @@ type CapabilityStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	Phase                 CapabilityPhase                        `json:"phase,omitempty"`
-	ServiceBindingName    string                                 `json:"serviceBindingName"`
-	ServiceBindingStatus  servicecatalogv1.ServiceBindingStatus  `json:"serviceBindingStatus"`
-	ServiceInstanceName   string                                 `json:"serviceInstanceName"`
-	ServiceInstanceStatus servicecatalogv1.ServiceInstanceStatus `json:"serviceInstanceStatus"`
+	Phase           CapabilityPhase   `json:"phase,omitempty"`
+	PodName         string            `json:"podName,omitempty"`
+	DatabaseName    string            `json:"databaseName"`
+	DatabaseStatus  string            `json:"databaseStatus"`
```

Comment:
Anything that is specific to a particular type of capability shouldn't appear in the generic status.

Reply:
I agree. I dont think that `DatabaseName`, `DatabaseStatus` are currently used. So we could remove them